developer-experience/recipes/execute-local-pipeline.md

```diff
@@ -0,0 +1,32 @@
+# Execute Local Pipeline
```

Comment:
Lets rename this recipe to indicate its specific to docker and containers. How about `execute-local-pipeline-with-docker.md`?